esx_service/vmodl/VsanDockerPersistentVolumeSystem.py

```diff
@@ -0,0 +1,389 @@
+# Copyright 2016 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Copyright 2016 VMware, Inc.  All rights reserved. 
+Licensed under the Apache License, Version 2.0 
+http://www.apache.org/licenses/LICENSE-2.0
+"""
+
+from VmodlDecorators import ManagedType, EnumType, Method, \
+   Return, RegisterVmodlTypes, F_OPTIONAL, Param, DataType, Attribute
+from pyVmomi import Vmodl
+from pyVmomi.VmomiSupport import newestVersions
+
+try:
+   from asyncVmodlEmitterLib import JavaDocs, Internal
+except ImportError:
+   pass
+   def JavaDocs(parent, docs):
+      def Decorate(f):
+         return f
+      return Decorate
+   def Internal(parent):
+      def Decorate(f):
+         return f
+      return Decorate
+
+
+# _VERSION = newestVersions.Get("vim")
+_VERSION = 'vim.version.version10'
+
+# Vmodl Names
+
+class VsanDockerPersistentVolumeTenantList:
+   _name = "vim.vsan.VsanDockerPersistentVolumeTenantList"
```

Comment:
Since the class is already defined in "vsan" namespace, why do we still add a "Vsan" prefix? Do we have any naming convention that mandates the "Vsan" prefix?